include/drivers/io/io_storage.h

```diff
@@ -22,6 +22,8 @@ typedef enum {
 	IO_TYPE_DUMMY,
 	IO_TYPE_FIRMWARE_IMAGE_PACKAGE,
 	IO_TYPE_BLOCK,
+	IO_TYPE_SPI,
+	IO_TYPE_EMMC,
```

Comment:
Do we need to add a new IO_TYPE_EMMC? I think IO_TYPE_BLOCK is good to eMMC device.